File: Python/tests/test_oms_bias.py
import pytest

from hbayesdm.models import oms_bias


def test_oms_bias():
    _ = oms_bias(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_oms_bias.R
context("Test oms_bias")
library(hBayesDM)

test_that("Test oms_bias", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(oms_bias(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
